tests/unittest_reporting.py

```diff
@@ -73,6 +76,55 @@ def test_parseable_output_regression():
     )
 
 
+def test_multi_format_output():
+    text = StringIO()
+    json = mktemp()
+    formats = ",".join(
+        [
+            "json:" + json,
+            "text",
```

Comment:
```suggestion
            "text"
```
In order to have a shorter black formatting (this magic comma force black to make multiple line)